src/core_plugins/kbn_vislib_vis_types/public/controls/vislib_basic_options.html

```diff
@@ -1,18 +1,19 @@
 <div>
   <div class="vis-option-item">
-    <label>
+    <label id="visualizeBasicSettingsLegendPosition">
       Legend Position
     </label>
     <select
+      aria-labelledby="visualizeBasicSettingsLegendPosition"
```

Comment:
In this case, I would prefer giving the `select` an `id` and the `label` a `for` attribute with that value.

That way all users would benefit, since the label is also outside the accessibility (i.e. in the regular DOM) linked to the select box, and you can e.g. click on the label to jump to the input (not working for select, but that's another topic).